### DISCUSSION TITLE
Rolling restart kafka or zookeeper managed by sps on demand

### DISCUSSION THREAD

**Discussion by elvis-cai** [strimzi/strimzi-kafka-operator#9052] [General]
Dear friend, we are using StrimziPodSet for kafka & zookeeper, somethings we want to change the pod node pool manually using kubectl cordon command, just wondering if there's any try to trigger a manual rolling restart for each of them, trying to search doc, I can find kafkaconnetor could be restart with `kubectl annotate KafkaConnector <connector-name> strimzi.io/restart=true`, however no luck for sps so far, thanks in advance. 🙏

**Comment by scholzj** [comment]
There are manual rolling update annotations you can use for Kafka and ZooKeeper nodes. You can find the exact name of the annotation in the docs - I think it is `stirmzi.io/manual-rolling-update`, but not 100% sure.

**Comment by elvis-cai** [comment]
Thanks @scholzj for the prompt reponse 👍 , found it [here](https://github.com/strimzi/strimzi-kafka-operator/blob/ca03d85567812d76e04873bbf0fe387412fb104b/documentation/modules/managing/proc-manual-rolling-update-strimzipodset.adoc#L27) using 
```
kubectl annotate strimzipodset <cluster_name>-kafka strimzi.io/manual-rolling-update=true

kubectl annotate strimzipodset <cluster_name>-zookeeper strimzi.io/manual-rolling-update=true
```
